lib/Optimizer/GraphOptimizer/ConstantFolding.cpp

```diff
@@ -302,6 +302,12 @@ bool glow::ConstantFold::run(Function *F, const CompilationContext &cctx) {
     if (!hasNonConstantOperationUser(N, *backend)) {
       continue;
     }
+    // Don't try to constant fold Nodes that are not supported by the
+    // Interpreter. These are usually Backend specific.
```

Comment:
```suggestion
    // Interpreter. These are usually backend-specific.
```

Reply:
I dunno, they're specific to a `Backend` object aren't they?